src/middlewares/ErrorHandler.ts

```diff
@@ -0,0 +1,19 @@
+import { 
+    Middleware,
+    ExpressErrorMiddlewareInterface
+} from "routing-controllers";
+
+@Middleware({ type: "after"})
+export class ErrorHandler implements  ExpressErrorMiddlewareInterface{
+    public error(
+        error: any,
+        request: any,
+        response: any,
+        next: (err: any) => any 
+    ) {
+        if (response.headersSent){
+            return;
+        }
+        response.json(error);
```

Comment:
You have to call `next()` at end of the function, because If you don't call next(), the request will be stuck at the current middleware, and subsequent middleware functions won't be executed. So, calling next() is crucial for the flow of control in Express middleware.

Reply:
Please read about [ExpressErrorMiddlewareInterface](https://github.com/typestack/routing-controllers#error-handlers), and use next() in a proper way.